src/Evolution/Executables/GrMhd/ValenciaDivClean/EvolveValenciaDivCleanBlastWave.hpp

```diff
@@ -0,0 +1,173 @@
+  // using boundary_condition_tag = analytic_solution_tag;
+  using analytic_variables_tags =
+      typename system::primitive_variables_tag::tags_list;
+  using equation_of_state_tag = hydro::Tags::EquationOfState<
+      typename analytic_data_tag::type::equation_of_state_type>;
+  using normal_dot_numerical_flux = OptionTags::NumericalFluxParams<
+      dg::NumericalFluxes::LocalLaxFriedrichs<system>>;
+  using limiter = OptionTags::SlopeLimiterParams<
+      SlopeLimiters::Minmod<3, system::variables_tag::tags_list>>;
+  using step_choosers =
+      tmpl::list<StepChoosers::Register::Cfl<3, Frame::Inertial>,
+                 StepChoosers::Register::Constant,
+                 StepChoosers::Register::Increase>;
+  using ordered_list_of_primitive_recovery_schemes = tmpl::list<
+      grmhd::ValenciaDivClean::PrimitiveRecoverySchemes::NewmanHamlin,
+      grmhd::ValenciaDivClean::PrimitiveRecoverySchemes::PalenzuelaEtAl>;
+
+  using compute_rhs = tmpl::flatten<tmpl::list<
+      Actions::ComputeVolumeFluxes,
+      dg::Actions::SendDataForFluxes<EvolutionMetavars>,
+      Actions::ComputeVolumeSources, Actions::ComputeVolumeDuDt,
+      // dg::Actions::ImposeDirichletBoundaryConditions<EvolutionMetavars>,
+      dg::Actions::ReceiveDataForFluxes<EvolutionMetavars>,
+      tmpl::conditional_t<local_time_stepping, tmpl::list<>,
+                          dg::Actions::ApplyBoundaryFluxesGlobalTimeStepping>,
+      Actions::RecordTimeStepperData>>;
+
+  using update_variables = tmpl::flatten<tmpl::list<
+      tmpl::conditional_t<local_time_stepping,
+                          dg::Actions::ApplyBoundaryFluxesLocalTimeStepping,
+                          tmpl::list<>>,
+      Actions::UpdateU, SlopeLimiters::Actions::SendData<EvolutionMetavars>,
+      SlopeLimiters::Actions::Limit<EvolutionMetavars>,
+      VariableFixing::Actions::FixVariables<VariableFixing::FixConservatives>,
+      Actions::UpdatePrimitives>>;
+
+  struct EvolvePhaseStart;
+  using component_list = tmpl::list<DgElementArray<
+      EvolutionMetavars, grmhd::ValenciaDivClean::Actions::Initialize<3>,
+      tmpl::flatten<tmpl::list<
+          VariableFixing::Actions::FixVariables<
+              VariableFixing::FixToAtmosphere<thermodynamic_dim>>,
+          Actions::UpdateConservatives,
+          SelfStart::self_start_procedure<compute_rhs, update_variables>,
+          Actions::Label<EvolvePhaseStart>, Actions::AdvanceTime,
+          VariableFixing::Actions::FixVariables<
+              VariableFixing::FixToAtmosphere<thermodynamic_dim>>,
+          Actions::UpdateConservatives, Actions::FinalTime,
+          tmpl::conditional_t<local_time_stepping,
+                              Actions::ChangeStepSize<step_choosers>,
+                              tmpl::list<>>,
+          compute_rhs, update_variables, Actions::Goto<EvolvePhaseStart>>>>>;
+
+  using const_global_cache_tag_list =
+      tmpl::list<analytic_data_tag,
```

Comment:
This should probably be in `Initialize::const_global_cache_tag_list` similar to how `normal_dot_numerical_fluxes_tag` is handled.
But `DgElementArray` does not read that list from the initialize action. So maybe we should open an issue for this.

Reply:
I filed issue #1191. Please feel free to comment there if my description of what should be done is insufficient :)